dns-resolver/rootfs/etc/unbound/unbound.tmpl

```diff
@@ -0,0 +1,114 @@
+server:
+  interface: 0.0.0.0
+  access-control: 0.0.0.0/0 allow
+  verbosity: 1
+
+  use-syslog: no
+  logfile: ""
+  
+  # port to answer queries from
+  port: 53
+
+  # Enable IPv4, "yes" or "no".
+  do-ip4: yes
+
+  # Enable IPv6, "yes" or "no".
+  do-ip6: yes
+
+  # Enable UDP, "yes" or "no".
+  do-udp: yes
+
+  # Enable TCP, "yes" or "no". If TCP is not needed, Unbound is actually
+  # quicker to resolve as the functions related to TCP checks are not done
+  do-tcp: yes
+
+  # Read  the  root  hints from this file. Default is nothing, using built in
+  # hints for the IN class. The file has the format of  zone files,  with  root
+  # nameserver  names  and  addresses  only. The default may become outdated,
+  # when servers change,  therefore  it is good practice to use a root-hints
+  # file.  get one from ftp://FTP.INTERNIC.NET/domain/named.cache
+  # root-hints: "/var/unbound/etc/root.hints"
+
+  # Use 0x20-encoded random bits in the query to foil spoof attempts.
+  # http://tools.ietf.org/html/draft-vixie-dnsext-dns0x20-00
+  # While upper and lower case letters are allowed in domain names, no significance
+  # is attached to the case. That is, two names with the same spelling but
+  # different case are to be treated as if identical. This means calomel.org is the
+  # same as CaLoMeL.Org which is the same as CALOMEL.ORG.
+  use-caps-for-id: yes
+
+  # the time to live (TTL) value lower bound, in seconds. Default 0.
+  # If more than an hour could easily give trouble due to stale data.
+  cache-min-ttl: 3600
```

Comment:
Can we set a per-forward-one ttl ?
1 hour could mean that we are still serving DNS A records, which have been long deleted..